### DISCUSSION TITLE
openaptx and bluez-alsa

### DISCUSSION THREAD

**Discussion by alexjaw** [Arkq/bluez-alsa#429] [Q&A] [answered]
Is it possible to use the aptx code from https://github.com/pali/libopenaptx (used by pulseaudio, https://lists.freedesktop.org/archives/pulseaudio-discuss/2019-February/030849.html) with bluez-alsa?

Cheers,
AJ

**Comment by arkq** [comment, marked as answer]
Yes, it is possible to use this library. Configure build like this:
```sh
../configure --enable-aptx --enable-aptx-hd --with-libopenaptx
```
